src/features/island/hud/components/settings-menu/SettingsMenu.tsx

```diff
@@ -186,6 +193,11 @@ export const SettingsMenu: React.FC<Props> = ({ show, onClose, isFarming }) => {
                       <span>Discord</span>
                     </Button>
                   </li>
+                  <li className="p-1">
```

Comment:
Can we put this inside the nested "Settings"?

My understanding is that players will learn about "Lost and Found" through Docs or Discord direction when they ask the question.

I'm not sure if we need to give this prime real-estate if it is only for a subset of users.

<img width="567" alt="Screen Shot 2023-02-13 at 4 45 16 pm" src="https://user-images.githubusercontent.com/11745561/218380211-e2049109-3185-4286-960a-bbe3bce93e01.png">